.dockerignore

```diff
@@ -0,0 +1,51 @@
+**/.DS_Store/**/*
```

Comment:
The Linux build is based on Docker.

Reply:
Is it possible to use the package build by `make dist` rather than copying? I do this for running docker builds for local testing (see [Makefile.am](https://github.com/cleishm/libcypher-parser/blob/master/Makefile.am#L28-L36)).

Reply:
No. The manylinux1 standard has restrictions on versions of symbols from glibc that are used so the c library has to be built inside docker to conform to these.

What is generated by `make dist`?

Reply:
Also, what do you mean by `source packaging for distribution`?

Reply:
The source tree from github contains the minimal source, and requires extra tools to get setup (autoconf, peg, etc).

Running `make dist` produces a source package that can be built on any system with minimal dependencies. I.e. it contains the generated configure script, the outputs from leg, etc.

This is the package that gets distributed in GitHub releases, and is the basis for all the various OS packaging (homebrew, debian, ubuntu, etc).

Reply:
Ok, I think I see your point. This is not a concern when linking statically, right?

Reply:
This isn't related to runtime or static linking. This is just the process by which distribution and packaging occur in autoconf projects. Basically it goes:

* Start with source checked out of version control (github)
* `autogen.sh && configure && make dist` => libcypher-parser-X.X.X.tar.gz
* Copy tarball to target environment: `tar xzf libcypher-parser-X.X.X.tar.gz && cd libcypher-parser-X.X.X && ./configure && make && make install` (or whatever is appropriate for packaging in that environment).

Reply:
Ok. Thanks for explanation.

Reply:
Done, the build in docker now uses source distribution.